src/view/edit-point-view.js

```diff
@@ -56,29 +59,46 @@ const createDestinationTemplate = (destinations, description) => {
       <p class="event__destination-description">${description}</p>
 
       <div class="event__photos-container">
-      <div class="event__photos-tape">
-      ${destinations.map(({ pictures: pictureForDescription }) => {
-      const [picture] = pictureForDescription;
-
-      return `<img class="event__photo" src="${picture.src}.jpg" alt="Event photo">`;
-    }).join('')}
-
+        <div class="event__photos-tape">
+          ${pictures.map(({ src }) => `<img class="event__photo" src="${src}.jpg" alt="Event photo">`).join('')}
         </div>
-       </div>
-      </section>
+      </div>
+    </section>
   `);
 };
 
+const createDestinationOptionsTemplate = (options) => {
+  if (!options?.length) {
+
+    return '';
+  }
+
+  return options.map((option) => `<option value=${option.name}></option>`).join('');
+};
+
+const createEventTypeListTemplate = (types, currentType) => types.map((type) => {
+  const checked = type === currentType ? 'checked' : '';
+
+  return (
+    `<div class="event__type-item">
+      <input id="event-type-${type}-1"
```

Comment:
Здесь единичку можно заменить на point.id, соответственно подправить лейбл
```suggestion
      <input id="event-type-${type}-${point.id}"
```